flang/lib/Lower/OpenMP/OpenMP.cpp

```diff
@@ -1062,11 +1062,11 @@ static void genSimdClauses(lower::AbstractConverter &converter,
   cp.processReduction(loc, clauseOps);
   cp.processSafelen(clauseOps);
   cp.processSimdlen(clauseOps);
-  // TODO Support delayed privatization.
+  cp.processAligned(clauseOps);
```

Comment:
Nit: Sort alphabetically here as well (helps with consistency in the output MLIR) unless it needs to go after processing some other specific clause, in which case it should be documented.